superset/views/utils.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from collections import defaultdict
+
+from flask import g
+from flask_appbuilder.security.sqla import models as ab_models
+
+from superset import db
+
+
+def bootstrap_user_data(given_username=None):
```

Comment:
`given_username` is confusing to me as it sounds like "Given Name" which means something else. It would be ok to receive `username` and to go `username = username or g.user.username` on the first line